src/main/java/uk/gov/hmcts/juror/api/moj/service/JurorRecordServiceImpl.java

```diff
@@ -1214,6 +1225,73 @@ private List<JurorAttendanceDetailsResponseDto.JurorAttendanceResponseData> getA
             .collect(Collectors.toList());
     }
 
+    @AllArgsConstructor
+    @NoArgsConstructor
+    @Getter
+    private static class PaymentSummaryData {
+        BigDecimal financialLoss = BigDecimal.ZERO;
+        BigDecimal travel = BigDecimal.ZERO;
+        BigDecimal subsistence = BigDecimal.ZERO;
+        BigDecimal paid = BigDecimal.ZERO;
+
+        PaymentSummaryData add(PaymentSummaryData target) {
+            financialLoss = financialLoss.add(target.getFinancialLoss());
+            travel = travel.add(target.getTravel());
+            subsistence = subsistence.add(target.getSubsistence());
+            paid = paid.add(target.getPaid());
+
+            return this;
+        }
```

Comment:
Can the target ever have null for financial loss / travel / substance / paid?

if so we may need to use BigDecimalUtil.getOrZero(...)

Reply:
This is the add function on PaymentSummaryData which, I guess it might be because there's a NoArgsConstructor. I'll ponder this.

Reply:
Ah no on the object it shouldn't ever be null as it should default to BigDecimal.ZERO, so I believe this is safe.